packages/react-fiche-service-public/src/components/ElementBuilder.js

```diff
@@ -22,11 +22,15 @@ const parseChildren = (children, headingLevel) => (
 
 // Beware, this one is recursive
 export function ElementBuilder({ data, headingLevel }) {
+  // Sometime we get elements without children...
+  if (!data) return null;
   // In case we get children
   if (Array.isArray(data)) {
-    return data.map((child, index) => (
-      <ElementBuilder key={index} data={child} headingLevel={headingLevel} />
-    ));
+    return data.map((child, index) => {
+      return (
```

Comment:
return not needed ;)

Reply:
Haha je crois que j'avais mis un console log juste au dessus avant de le retirer à la va vite !